dtfs-central-api/src/services/state-machines/utilisation-report/utilisation-report.state-machine.ts

```diff
@@ -93,6 +94,8 @@ export class UtilisationReportStateMachine {
             return handleUtilisationReportGenerateKeyingDataEvent(this.report, event.payload);
           case 'EDIT_PAYMENT':
             return handleUtilisationReportEditPaymentEvent(this.report, event.payload);
+          case 'REMOVE_PAYMENT_FEES':
```

Comment:
I think the event name could be slightly clearer e.g. `REMOVE_FEEs_FROM_PAYMENT_GROUP`

Reply:
Renamed thanks